src/components/common/ScaleBox.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="container">
+    <div class="scaling-wrapper">
+      <slot></slot>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      scale: 1
+    }
+  },
+  computed: {
+    containerStyle() {
+      return {
+        transform: `scale(${this.scale})`,
```

Comment:
Curious what problem this component is solving, without seeing it in action I'm wondering if you can solve this purely with CSS.

Reply:
This shouldn't have been committed since I found an alternate solution! Deleted.